commands/guild/twitch-announcer.js

```diff
@@ -0,0 +1,443 @@
+          'Rank:',
+          user.data[0].broadcaster_type.toUpperCase() + '!',
+          true
+        )
+        .setFooter(
+          Twitch_DB.get(message.guild.id).twitchAnnouncer.savedName,
+          Twitch_DB.get(message.guild.id).twitchAnnouncer.savedAvatar
+        )
+        .setTimestamp(Twitch_DB.get(message.guild.id).twitchAnnouncer.date);
+    }
+
+    //Disable Embed
+    const disabledEmbed = new MessageEmbed()
+      .setAuthor(
+        message.member.guild.name + ' Announcer Settings',
+        `https://static.twitchcdn.net/assets/favicon-32-d6025c14e900565d6177.png`,
+        'https://twitch.tv/' + user.data[0].display_name
+      )
+      .setTitle(`:x: Twitch Announcer Disabled!`)
+      .setColor('#6441A4')
+      .setThumbnail(user.data[0].profile_image_url)
+      .addField(
+        'Pre-Notification Message',
+        `${Twitch_DB.get(message.guild.id).twitchAnnouncer.botSay}`
+      )
+      .addField(
+        `Streamer`,
+        `${Twitch_DB.get(message.guild.id).twitchAnnouncer.name}`,
+        true
+      )
+      .addField(
+        `Channel`,
+        `${Twitch_DB.get(message.guild.id).twitchAnnouncer.channel}`,
+        true
+      )
+      .addField(
+        `Checking Interval`,
+        `***${
+          Twitch_DB.get(message.guild.id).twitchAnnouncer.timer
+        }*** minute(s)`,
+        true
+      )
+      .addField('View Counter:', user.data[0].view_count, true);
+    if (user.data[0].broadcaster_type == '')
+      disabledEmbed.addField('Rank:', 'BASE!', true);
+    else {
+      disabledEmbed
+        .addField(
+          'Rank:',
+          user.data[0].broadcaster_type.toUpperCase() + '!',
+          true
+        )
+        .setFooter(
+          Twitch_DB.get(message.guild.id).twitchAnnouncer.savedName,
+          Twitch_DB.get(message.guild.id).twitchAnnouncer.savedAvatar
+        )
+        .setTimestamp(Twitch_DB.get(message.guild.id).twitchAnnouncer.date);
+    }
+    //Check Post
+    if (textFiltered == 'check') {
+      if (Twitch_DB.get(message.guild.id).twitchAnnouncer.status == 'disable')
+        message.say(disabledEmbed);
+      else {
+        return message.say(enabledEmbed);
+      }
+      return;
+    }
+    //Disable Set
+    if (textFiltered == 'disable') {
+      Twitch_DB.set(`${message.guild.id}.twitchAnnouncer.status`, 'disable');
+      message.say(disabledEmbed);
+    }
+
+    //Enable Set
+    if (textFiltered == 'enable') {
+      Twitch_DB.set(`${message.guild.id}.twitchAnnouncer.status`, 'enable');
+      message.say(enabledEmbed);
+
+      //Ticker Section (Loop)
+      var Ticker = setInterval(async function() {
+        let statusCheck = Twitch_DB.get(message.guild.id).twitchAnnouncer
+          .status;
+        if (statusCheck == 'disable') {
+          clearInterval(Ticker);
+          return;
+        }
+
+        let announcedChannel = message.guild.channels.cache.find(
+          c => c.name == Twitch_DB.get(message.guild.id).twitchAnnouncer.channel
```

Comment:
change the 'c' parameter to 'channel', it's better practice

Reply:
Renamed to channel